### DISCUSSION TITLE
Can {{<toc>}} performanced better or being more structural?

### DISCUSSION THREAD

**Discussion by gitaeron** [Lednerb/bilberry-hugo-theme#536] [General]
![image](https://user-images.githubusercontent.com/87517605/232817859-675b3a1b-fcfe-4f17-b46d-43b627729e43.png)

It's just a simply ask( And I don't searched similar keyword in issue, sorry)

But I found out that only font size is the way to recognize the difference.. Is there a better way to shown that,  like a tree structure?

**Comment by igor-baiborodine** [comment]
@gitaeron, to display the TOC like a tree, you should use different header sizes, e.g., `## Header 2`, `### Header 3`, etc. 

Here is a test article on the Bilberry Sandbox: https://www.bilberry-sandbox.kiroule.com/article/toc-test-h2-h3-h4-h5/

And the corresponding content file: https://raw.githubusercontent.com/igor-baiborodine/bilberry-hugo-theme-sandbox/master/content/article/toc-test-h2-h3-h4-h5.md

For more details on how the TOC is implemented in the Bilberry Hugo theme, you can read this article: https://www.kiroule.com/article/add-table-of-contents-to-hugo-theme/

**Reply by gitaeron** [reply]
thanks, it helped me a lot.
